Vue CLIを使って、SPAなメモアプリを作成しました。

naomichi-h opened the pull request:
状態管理にVuexを使用しています。
レビューのほど、よろしくお願いいたします。

naomichi-h committed:
プロジェクト作成

naomichi-h committed:
memo appを作成しました

komagata commented on src/components/createMemo.vue:
```diff
@@ -0,0 +1,26 @@
+  padding: 8px 0;
+
+}
+
```
不要な空行があるみたいです〜

komagata commented on src/components/createMemo.vue:
```diff
@@ -0,0 +1,26 @@
+  name: 'CreateMemo',
+  methods: {
+    addMemo () {
+      this.$store.commit('editFlgOn')
```
> Flg

あまり略さない方がいいかもです〜

[プログラミング初心者は変数名やメソッド名を略さない方がいいよ、という話 - give IT a try](https://blog.jnito.com/entry/2020/10/20/092724)

またフラグという単語は意味が広すぎるのでなるべく使わずに表現できた方がわかりやすいと思います。

komagata commented on src/components/indexMemo.vue:
```diff
@@ -0,0 +1,69 @@
+<template>
+  <ul class="item-wrap">
+    <li class="item" v-for="(memo, index) in memos" v-bind:key="memo.id" @click="editMemoStart(memo, index)" v-bind:class="{ editActive: isActive(index) }">
+      {{ memo.text.split(/\r\n|\r|\n/)[0] }}
```
算出プロパティを使った方がいいかもです。

[算出プロパティとウォッチャ — Vue.js](https://jp.vuejs.org/v2/guide/computed.html#%E7%AE%97%E5%87%BA%E3%83%97%E3%83%AD%E3%83%91%E3%83%86%E3%82%A3)

naomichi-h committed:
fix:レビューを受けてコードの修正を行いました
- いくつかのファイルにあった不要な空行を削除し、スタイルを統一しました
- Flgという単語の使用をやめ、新たにModeに変更しました(editFlg -> editMode など)
- index.vueのメモタイトル出力部分で、マスタッシュ内に書いていた処理をcomputedに移動しました

naomichi-h commented:
レビューありがとうございます😄
コメントいただいた箇所の修正を行いましたので、確認のほどよろしくお願いいたします🙇‍♂️

komagata commented on .gitignore:
```diff
@@ -0,0 +1,23 @@
+node_modules
+/dist
+
+
```
不要な空行があるようです〜

komagata commented on src/App.vue:
```diff
@@ -0,0 +1,72 @@
+}
+</style>
+
+<style>
```
style要素が二つあるようです。
scopedでないstyleが必要であればcssファイルに書くのが良いと思います。

komagata commented on src/components/header.vue:
```diff
@@ -0,0 +1,15 @@
+<template>
+    <h1 class="app-title">memo app</h1>
```
indentの数が他と違うようです〜。

naomichi-h committed:
fix:レビューを受けて修正を行いました２
不要な空行およびインデントを削除しました
グローバルなstyleをcssファイルに切り出しました

naomichi-h commented:
レビューありがとうございます😄
コメントいただいた箇所の修正を行いましたので、確認のほどよろしくお願いいたします🙇‍♂️

komagata reviewed (approved):
確認しました、OKですー🙆‍♂️
